packages/lit-dev-content/site/docs/components/properties.md

```diff
@@ -264,14 +264,46 @@ When a property changes, the following sequence occurs:
 
 1.  The property's setter is called.
 1.  The setter calls the component's `requestUpdate` method.
-1.  The property's old and new values are compared. If the property has a `hasChanged` function, it's called with the property's old and new values.
+1.  The property's old and new values are compared.
+    -  By default Lit uses a strict inequality test to determine if the value has changed (that is `newValue !== oldValue`).
+    -  If the property has a `hasChanged` function, it's called with the property's old and new values.
 1.  If the property change is detected, an update is scheduled asynchronously. If an update is already scheduled, only a single update is executed.
 1.  The component's `update` method is called, reflecting changed properties to attributes and re-rendering the component's templates.
 
+Note that if you mutate an object or array property, it won't trigger an update, because the object itself hasn't changed. For more information, see [Mutating object and array properties](#mutatingobjectandarrayproperties).
```

Comment:
```suggestion
Note that if you mutate an object or array property, it won't trigger an update, because the object itself hasn't changed. For more information, see [Mutating object and array properties](#mutating-properties).
```

Reply:
Thanks!